test/deploy.sh

```diff
@@ -69,16 +68,25 @@ else
 	export INSTANCE_NAME="${INSTANCE_NAME:-${INSTANCE_NAME_DEFAULT}}"
 fi
 
+# Set extra parameters
+export OUTPUT="${ROOT}/_output/${INSTANCE_NAME}"
+export RESOURCE_GROUP="${INSTANCE_NAME}"
+export DEPLOYMENT_NAME="${INSTANCE_NAME}"
+
+source "${ROOT}/test/common.sh"
+
 make -C "${ROOT}"
+generate_template
+set_azure_account
 trap cleanup EXIT
-deploy
+deploy_template
 
 if [[ -z "${VALIDATE:-}" ]]; then
 	exit 0
 fi
 
-export SSH_KEY="${ROOT}/_output/${INSTANCE_NAME}/id_rsa"
-export KUBECONFIG="${ROOT}/_output/${INSTANCE_NAME}/kubeconfig/kubeconfig.${LOCATION}.json"
+export SSH_KEY="${OUTPUT}/id_rsa"
```

Comment:
It feels weird that we export these here and then also export them in `./test/step.sh`. If we want to split this script into smaller bites, then let's go ahead and yank out the call to validate here. (FWIW, in my opinion, you can make breaking changes to this -- I'm the only one who seems to be using it)